src/main/java/edu/wpi/grip/core/operations/opencv/MatFieldAccessor.java

```diff
@@ -0,0 +1,77 @@
+package edu.wpi.grip.core.operations.opencv;
+
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.InputSocket;
+import edu.wpi.grip.core.OutputSocket;
+import edu.wpi.grip.core.SocketHint;
+import org.bytedeco.javacpp.opencv_core.Mat;
+import org.bytedeco.javacpp.opencv_core.Size;
+
+public class MatFieldAccessor implements CVOperation{
+    private static final Mat defaultsMat = new Mat();
+    private final SocketHint matHint = new SocketHint("input", Mat.class, Mat::new, SocketHint.View.NONE);
+    private final SocketHint sizeHint = new SocketHint("size", Size.class, Size::new);
+    private final SocketHint isNullHint = new SocketHint("is null", Boolean.class, defaultsMat.isNull());
```

Comment:
Mats shouldn't ever be `null`.  If this returns true, I'm pretty sure something in the application would crash (or more likely OpenCV would segfault), so having this as an output is not useful.